direct/io-bulk-fs/src/main/java/cz/o2/proxima/direct/bulk/BinaryBlobFormat.java

```diff
@@ -237,75 +244,62 @@ private StreamElement fromBytes(byte[] data) throws IOException {
     }
 
     @Override
-    public void close() throws IOException {
+    public void close() {
       if (blobStream != null) {
-        blobStream.close();
+        ExceptionUtils.unchecked(blobStream::close);
```

Comment:
nit: Maybe we should have our own version of `Closeables.close(Closeable)` instead. (the guava one rethrows the exception 🤦‍♂ )

Reply:
I think this is readable enough. The quava version might be `Closeables.closeQuietly(Closeable)`, but that is nearly as verebose as this.